### PR TITLE
feat(SegmentedControls): add pre-selected states for fit and fixed 

### PR DESCRIPTION
**Pull request by aammami-ledger**
[DLS-802](https://ledgerhq.atlassian.net/browse/DLS-802)

[DLS-802]: https://ledgerhq.atlassian.net/browse/DLS-802?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

Screenshot 2026-06-10 at 14 48 11

### CODE REVIEW

File: libs/ui-rnative/src/lib/Components/SegmentedControl/usePillLayout.ts [hunk 1]
```diff
@@ -78,31 +79,38 @@ export function usePillLayout({
...
   );
 
   useEffect(() => {
     if (!hasLayoutRef.current) return;
 
+    const skipAnimation = !animatePillRef.current;
+    if (skipAnimation) {
+      animatePillRef.current = true;
+    }
+    const config = skipAnimation ? { duration: 0 } : timingConfig;
```
[the remark this answers is not in the document]
Comment (aammami-ledger) [reply libs/ui-rnative/src/lib/Components/SegmentedControl/usePillLayout.ts:101]: It's an edge case that should not take place as we always know the values of the buttons

**Review by zel-kass — APPROVED** [review]
Besides the test coverage, this looks nice ! gg 🤓
